### PR TITLE
PAPP-23257: removing biased language from the action name

### PR DESCRIPTION
**Pull request by dgopani-crest**
As discussed with @mattsayar-splunk on slack, for now, we do not need to release the app with those changes. We will remove a biased name from the app and merge MR into the next. 
We will release the app in the future when contributors submit the new code changes.
Not fixing other static job failures as it is related to py2 and we are not going to release this app.

### CODE REVIEW

File: aws_connector.py [hunk 1]
```diff
@@ -1219,11 +1220,12 @@ def finalize(self):
 if __name__ == '__main__':
 
     import sys
+
     import pudb
     pudb.set_trace()
 
     if (len(sys.argv) < 2):
```
Comment (pdesai-crest) [inline aws_connector.py:1227]: redundant parantheses

File: aws_connector.py [hunk 2]
```diff
@@ -1219,11 +1220,12 @@ def finalize(self):
...
-        print "No test json specified as input"
+        print("No test json specified as input")
         exit(0)
```
Comment (pdesai-crest) [inline aws_connector.py:1229]: can we add sys.exit?

**Review by pdesai-crest — DISMISSED** [review]

**Comment by pdesai-crest** [comment]
please fix static tests pipeline

File: aws.json [hunk 3]
```diff
@@ -180,14 +179,14 @@
...
+            "action": "unblock ip",
+            "description": "Unblock IP by removing any block rules from NACLs",
             "type": "correct",
-            "identifier": "whitelist_ip",
+            "identifier": "unblock_ip",
             "read_only": false,
             "parameters": {
                 "ip": {
-                    "description": "IP address to whitelist",
+                    "description": "IP address to Unblock",
```
Comment (ngaglani-crest) [inline aws.json]:
```suggestion
                    "description": "IP address to unblock",
```

**Review by pdesai-crest — APPROVED** [review]

**Review by ngaglani-crest — APPROVED** [review]

File: aws_connector.py (hunk 2 above)
Comment (ngaglani-crest) [inline aws_connector.py:1229]:
```suggestion
        sys.exit(0)
```

**Pull request merged**